containers/Ideas/Idea.js

```diff
@@ -18,7 +19,12 @@ export const Idea = (props) => {
   };
   const toggleMenu = () => setMenuVisible(!menuVisible);
   const [menuVisible, setMenuVisible] = useState(false);
-  const menuItems  = [
+  const toggleDeleteModal = () => (
```

Comment:
i know i started this mess :) (changed it in a pending PR), but lets try to have an order in our react function components.

E.g. I would start with variable and state definitions, followed by function definitions. Maybe something like the following?

`const {params, createdDate} = props.route.params;`
`const {menuVisible, setMenuVisible} = useState(false);`
`const {deleteModalVisible, setDeleteModalVisible} = useState(false);`
`const toggleMenu = () => ....`
`const toggleDeleteModal = () => ....`

Reply:
Ah, yes! I like rules I can stick to. :)